src/TwigEnvironmentFactory.php

```diff
@@ -245,11 +239,13 @@ private function loadRuntimeLoader($runtimeLoader, ContainerInterface $container
         }
 
         if (! $runtimeLoader instanceof RuntimeLoaderInterface) {
-            throw new Exception\InvalidRuntimeLoaderException(sprintf(
-                'Twig runtime loader must be an instance of %s; "%s" given,',
-                RuntimeLoaderInterface::class,
-                is_object($runtimeLoader) ? get_class($runtimeLoader) : gettype($runtimeLoader)
-            ));
+            throw new Exception\InvalidRuntimeLoaderException(
+                sprintf(
+                    'Twig runtime loader must be an instance of %s; "%s" given,',
+                    RuntimeLoaderInterface::class,
+                    is_object($runtimeLoader) ? get_class($runtimeLoader) : gettype($runtimeLoader)
+                )
+            );
```

Comment:
Is this change dictated by our coding standard? Original formatting should be just fine, I think.

Reply:
I just ran cs-fix across the project, either that or PhpStorm did it, though I typically leave settings as default for mezzio/laminas repos.

Reply:
fix incoming